03_Spatial_Analyst/Spatial_Analyst.Rmd

```diff
@@ -134,10 +137,9 @@ Let's start by computing the _slope_ of the **swissAlti3d** raster dataset.
 ```{r}
 # Importing a raster dataset
 # swissALTI3D2019.tif -- Source: https://geovite.ethz.ch
```

Comment:
if the source is from geovite, are you sure we are allowed to distribute it in this manner?

Reply:
I do not know per se, but the same dataset is also available in geodata2use. We could also have this as source

Reply:
geovite and geodata2use have restrict the distribution of data. Normally, we would need to replace this dataset at one point, but swissalti3d is going to be freely available as of 1.3.2021 it think